app/extras/queries/visible_discussions.rb

```diff
@@ -43,6 +43,14 @@ def join_to_motions
     end
   end
 
+  def join_to_starred_motions
```

Comment:
Ah, it's a thread with an active proposal and a star, which we want to
prioritize over starred threads and threads with active proposals.

On Mon, Aug 3, 2015 at 2:10 AM, Robert Guthrie <notifications@github.com>
wrote:

> In app/extras/queries/visible_discussions.rb
> <https://github.com/loomio/loomio/pull/2369#discussion_r36067595>:
>
> > @@ -43,6 +43,14 @@ def join_to_motions
> >      end
> >    end
> >
> > +  def join_to_starred_motions
>
> What is a starred motion? ... seems a little peculiar
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2369/files#r36067595>.
>